addons/hr_contract/views/hr_contract_views.xml

```diff
@@ -297,16 +298,25 @@
                     </t>
                     <t t-name="kanban-box">
                         <div class="oe_kanban_card oe_kanban_global_click">
+                            <div class="oe_kanban_header">
+                                <span class="float-end mr4" style="padding-right: 5px; padding-top: 5px;">
```

Comment:
i would use classes for styling here instead, `class="float-end mr4 p-1`

Reply:
Thx! I didn't knew about that!

Reply:
Thx! I didn't knew about that!